sdk/program/src/vote/instruction.rs

```diff
@@ -185,6 +185,20 @@ impl VoteInstruction {
             _ => panic!("Tried to get slot on non simple vote instruction"),
         }
     }
+
+    /// Only to be used on vote instructions (guard with is_simple_vote),  panics otherwise
```

Comment:
Just to clarify that is_simple_vote guards outside of this function, in another words, non siple_vote transaction would never land here?

Reply:
ah right, i just copy pasted from `last_voted_slot()` above, i'll add the `assert!`. This should only be used on simple vote transactions.